src/app/components/RegisterValidatorHome/RegisterValidatorHome.tsx

```diff
@@ -29,39 +27,27 @@ const RegisterValidatorHome = () => {
   };
 
   return (
-    <Paper className={classes.mainContainer}>
-      <BackNavigation to={config.routes.HOME} text={translations.HOME.TITLE} />
-      <Header title={translations.VALIDATOR.HOME.TITLE} subtitle={translations.VALIDATOR.HOME.DESCRIPTION} />
-      <br />
-      <Grid container wrap="nowrap" spacing={3} className={classes.rowGridContainer}>
-        <Grid item xs={6} md={6} zeroMinWidth className={classes.rowGridContainer}>
-          <ConditionalLink to={config.routes.VALIDATOR.CREATE} condition onClick={() => redirectTo(config.routes.VALIDATOR.CREATE)}>
-            <Paper>
-              <Grid container wrap="nowrap" className={classes.bigSquareButton}>
-                <Grid item md={12} xs={12} className={classes.bigSquareButtonGrid}>
-                  <img src="/images/etherium.png" alt="Create Validator" className={classes.bigSquareButtonIcon} />
-                  <Typography noWrap variant="h6" className={classes.guideStepText}>Create Validator</Typography>
-                </Grid>
-              </Grid>
-            </Paper>
-          </ConditionalLink>
-        </Grid>
+    <Screen
+      navigationText={translations.HOME.TITLE}
+      navigationLink={config.routes.HOME}
+      title={translations.VALIDATOR.HOME.TITLE}
+      subTitle={translations.VALIDATOR.HOME.DESCRIPTION}
+      body={(
+        <Grid container wrap="nowrap" spacing={3} className={classes.columnGridContainer}>
+          <Grid item xs={12} md={12} zeroMinWidth className={classes.columnGridContainer}>
+            <ConditionalLink to={config.routes.VALIDATOR.CREATE} condition onClick={() => redirectTo(config.routes.VALIDATOR.CREATE)}>
+              <LinkButton text1={'Create Validator'} text2={''} />
```

Comment:
What is text1 and text2, can we name these props with better sense?
Like, `primaryLabel`, `secondaryLabel`